webapp/app/elster_client/elster_client.py

```diff
@@ -52,13 +52,22 @@ def send_to_erica(*args, **kwargs):
             logger.info(f'Completed Erica POST request with args {args!r}, got code {response.status_code}')
         return response
     except Timeout as error:
-        logger.info(f'Erica POST request raised a timeout exception, got code {error.response.status_code}')
+        # status code is 0 if response of the error is none
+        status_code = 0 if error.response is None else error.response.status_code
+        
+        logger.info(f'Erica POST request raised a timeout exception, got code {status_code}')
```

Comment:
![image](https://user-images.githubusercontent.com/91742583/151406466-4d079fc8-ebd9-4394-ade6-e18a30f90cc8.png)

Timeout / ConnectionTimeout / ... are RequestExceptions and may have a response. We should log the status_code (maybe 408?) because here we can see that the client timeout of our webapp to erica is to short maybe? we should wait for the timeout on the server side to handle it properly

Reply:
Or we should use a nice async job management :smiley_cat:

Reply:
I'm wrong with the Timeout. If we get a 408 is client side - the server didn't respond in the expected time. So your right we should never get a response if a Timeout occurs. I was confused about the Timeout -> RequestException and need of a useless field of response for that kind of error...

Reply:
The same for ConnectionError - it's a client side error.